src/shared/model/Routes.ts

```diff
@@ -100,6 +100,7 @@ export type ApiRoutePaths =
 	| '/user/change-email'
 	| '/unsubscribe'
 	| '/subscribe'
+	| '/user'
```

Comment:
what is this for ?

Reply:
For the IDAPI `GET /user?emailAddress=` call.